### PR TITLE
Feature/light mode

### PR DESCRIPTION
**Pull request by EneOne**
Please check!

### CODE REVIEW

**Review by NWisotzki — APPROVED** [review]

File: SkillMatrix/ClientApp/src/app/admin/skills/services/skill.service.ts [hunk 1]
```diff
@@ -21,6 +21,16 @@ export class SkillService {
 
   constructor(private http: HttpClient) { }
 
+  //HERE
```
Comment (conradoalexsander) [inline SkillMatrix/ClientApp/src/app/admin/skills/services/skill.service.ts:24]: There? Ahhaha

File: SkillMatrix/ClientApp/src/app/admin/skills/services/skill.service.ts [hunk 2]
```diff
@@ -21,6 +21,16 @@ export class SkillService {
...
+  //HERE
```
Comment (conradoalexsander) [inline SkillMatrix/ClientApp/src/app/admin/skills/services/skill.service.ts:25]:
```suggestion
```

File: SkillMatrix/ClientApp/src/app/admin/skills/services/skill.service.ts [hunk 3]
```diff
@@ -21,6 +21,16 @@ export class SkillService {
...
+  //HERE
+  //HERE
+  //HERE
+  //HERE
+  //HERE
+  //HERE
+  //HERE
```
Comment (conradoalexsander) [inline SkillMatrix/ClientApp/src/app/admin/skills/services/skill.service.ts:32]:
```suggestion
```

File: SkillMatrix/ClientApp/src/app/admin/skills/services/skill.service.ts (hunk 1 above)
Comment (conradoalexsander) [inline SkillMatrix/ClientApp/src/app/admin/skills/services/skill.service.ts:24]:
```suggestion
```

**Review by conradoalexsander — DISMISSED** [review]

File: SkillMatrix/ClientApp/src/app/admin/skills/services/skill.service.ts (hunk 2 above)
Comment (EneOne) [reply SkillMatrix/ClientApp/src/app/admin/skills/services/skill.service.ts:25]: Presence is more than just being there.
Malcolm Forbes

**Review by NWisotzki — DISMISSED** [review]

**Review by conradoalexsander — APPROVED** [review]

**Review by NWisotzki — APPROVED** [review]

**Pull request merged**